pom.xml

```diff
@@ -136,7 +136,7 @@
         <junit-platform-surefire-provider.version>1.3.2</junit-platform-surefire-provider.version>
         <opentest4j.version>1.2.0</opentest4j.version>
         <gson.version>2.8.2</gson.version>
-        <vertx.kafka.client>3.9.1</vertx.kafka.client>
+        <vertx.kafka.client>4.0.3</vertx.kafka.client>
         <exec-maven-plugin.version>1.6.0</exec-maven-plugin.version>
         <netty.version>4.1.60.Final</netty.version>
```

Comment:
Could you please try to bump Netty to 4.1.62.Final and see if it works with Vert.x 4.0.3?

Reply:
Does not work even with 4.1.63.Final.

Reply:
So ... does it work at least with the 4.1.60 which is default in VErt.x 4.0.3?

Reply:
Yes

Reply:
Then I guess that is the way to go ... and hopefully Vert.x release another fix later to work with the fixed Netty 🤷‍♂️